projects/landing-page/src/index.html

```diff
@@ -8,10 +8,20 @@
   <body>
     <header class="app-header">
       <div class="app-header__content">
-        <picture class="app-header__logo">
-          <img src="assets/images/headsup-logo.png" width="56" height="56" alt="Heads up! Logo">
-        </picture>
-        <h1 class="app-header__product-title">Heads up!</h1>
+        <div class="app-header__title">
+          <picture class="app-header__logo">
+            <img src="assets/images/headsup-logo.png" width="56" height="56" alt="Heads up! Logo">
+          </picture>
+          <h1 class="app-header__product-title">Heads up!</h1>
+        </div>
+        <a
+          class="app-header__github-link"
+          target="_blank"
+          rel="nofollow"
+          href="https://github.com/voorhoede/heads-up"
+        >
+          <img src="assets/images/github-logo.svg" alt="Github Logo" />
+        </a>
```

Comment:
I liked the way it look on the header more than all the way down, plus looking at other open source projects it's normal to have it there.